packages/grid/_modules_/grid/models/colDef/gridBooleanColDef.tsx

```diff
@@ -4,6 +4,14 @@ import { renderBooleanCell } from '../../components/cell/GridBooleanCell';
 import { renderEditBooleanCell } from '../../components/cell/GridEditBooleanCell';
 import { gridNumberComparer } from '../../utils/sortingUtils';
 import { getGridBooleanOperators } from './gridBooleanOperators';
+import { GridValueFormatterParams } from '../params/gridCellParams';
+
+function gridBooleanFormatter({ value }: GridValueFormatterParams) {
```

Comment:
I think for now we can keep that one here. If in future we need it in more than 1 place we can move to the `/utils`.